src/components/notification/notification.tsx

```diff
@@ -0,0 +1,62 @@
+import { CSSProperties, FC, ReactElement, useEffect } from 'react';
+import { NotificationProps, NotificationType } from '@components/notification/types';
+import styles from './notification.module.scss';
+import classNames from 'classnames/bind';
+import { CloseIcon, ErrorIcon, InfoIcon, SuccessIcon } from '@/components';
+
+const cx = classNames.bind(styles);
+const ERROR_DURATION = 7000;
+const DEFAULT_DURATION = 4000;
+
+export const Notification: FC<NotificationProps> = ({
```

Comment:
This component name as `SystemAlert` in design system.
Please clarify the final naming with UX as we agreed to follow the common naming convention for components from the Design system.